drift_dev/lib/src/cli/commands/make_migrations.dart

```diff
@@ -0,0 +1,505 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:drift_dev/src/analysis/results/results.dart';
+import 'package:drift_dev/src/cli/cli.dart';
+import 'package:drift_dev/src/cli/commands/schema.dart';
+import 'package:drift_dev/src/cli/commands/schema/generate_utils.dart';
+import 'package:drift_dev/src/cli/commands/schema/steps.dart';
+
+import 'package:drift_dev/src/services/schema/schema_files.dart';
+import 'package:io/ansi.dart';
+import 'package:path/path.dart' as p;
+
+class MakeMigrationCommand extends DriftCommand {
+  MakeMigrationCommand(super.cli);
+
+  @override
+  String get description => """
+Generates migrations utilities for drift databases
+
+### Usage
+After defining your database for the first time, run this command to save the schema.
+When you are ready to make changes to the database, alter the schema in the database file, bump the schema version and run this command again.
+This will generate the following:
+
+1. A steps file which contains a helper function to write a migration from one version to another.
+
+  Example: 
```

Comment:
Remove the trailing space here and in line 49:

```suggestion
  Example:
`` `